google/cloud/spanner/samples/samples.cc

```diff
@@ -3287,6 +3231,43 @@ int RunOneCommand(std::vector<std::string> argv) {
   return 0;
 }
 
+std::string PickConfig(google::cloud::spanner::InstanceAdminClient client,
+                       std::string const& project_id) {
+  // Skip non-US configs. They are too slow.
+  std::string const included_prefix = "regional-us-";
+  // Exclude the US configs where we keep most of our test instances.
+  std::vector<std::string> const excluded{"regional-us-central1",
+                                          "regional-us-east1"};
+
+  std::string const config_prefix =
+      "projects/" + project_id + "/instanceConfigs/";
+
+  // Use a reservoir sampler of size k==1 to pick a region at random.
+  std::mt19937_64 generator(std::random_device{}());
```

Comment:
nit: this is only called from a function that already has a generator, why not pass it in?

Reply:
Done.  Thanks.